### DISCUSSION TITLE
Understanding tags & reporting on tags

### DISCUSSION THREAD

**Discussion by Normanras** [almarklein/timetagger#349] [Q&A] [answered]
Hey there @almarklein - once again, amazing app. I've been running it since September 2022 and looking back over all my tasks has been fascinating. I've mostly been using it for my client-facing tasks, but I've begun including tags on what language I'm working in, what sort of project I'm doing and more. Super useful! 

I've recently been leveraging multiple tags (as opposed to descriptions) and I read through the docs to ensure I'm using them correctly. I might have read the docs wrong, so I just wanted to confirm something. Here's how I'm currently using tags is `client`-`medium`-`project`. For instance:

Entry 1: #github #email #db_cleaning
Entry 2: #microsoft #meeting #implementation

When I look at my reporting, it shows me the total time for each entry with all the tags. Is it possible to see the reporting for _each_ tag? So I'd be curious to see my total time in meetings irrespective of the `client` or `project`, or total time with the `client`, etc. I hope that makes sense. 

And if this is not possible, is there a better way I can use the individual tags that I missed? Happy to adjust my workflow as I'm still perfecting my timetagger usage. Thanks again for a wonderful app!

**Comment by almarklein** [comment, marked as answer]
Hi @Normanras! You can always click on an individual tag in the overview on the right. That will select that tag and show the total time spend on it, as well as a break-down how it's used in combination with other tags.

Also see https://github.com/almarklein/timetagger/issues/244, which proposes an option to view the total time spend on individual tags.

**Comment by Normanras** [comment]
Awesome, @almarklein !! Cant wait for the next release :)
